### ISSUE TITLE
PooledNormals: Import fails when value from LIMS is null

### ISSUE THREAD

**Issue by allanbolipata** [mskcc/beagle#141]
Some records don't have any pooled normals - import fails because it's trying to iterate over `NoneType`.

A couple request Ids to check: `10640`, `10720`, etc.

I believe it's at this line: https://github.com/mskcc/beagle/blob/develop/beagle_etl/jobs/lims_etl_jobs.py#L107

Attached example error:

```
[2020-02-17 18:48:53,203: WARNING/ForkPoolWorker-3] File "/home/bolipatc/dev_voyager/beagle/beagle_etl/tasks.py", line 72, in process
    self._process()
[2020-02-17 18:48:53,204: WARNING/ForkPoolWorker-3] File "/home/bolipatc/dev_voyager/beagle/beagle_etl/tasks.py", line 97, in _process
    children = func(**self.job.args)
[2020-02-17 18:48:53,205: WARNING/ForkPoolWorker-3] File "/home/bolipatc/dev_voyager/beagle/beagle_etl/jobs/lims_etl_jobs.py", line 108, in fetch_samples
    for f in pooled_normals:
[2020-02-17 18:48:53,212: INFO/ForkPoolWorker-3] Job 2e238ac1-0bf9-4d19-ac07-d4b7369357e1 in status: FAILED
```

**Comment by allanbolipata** [comment]
Applied potential fix here https://github.com/mskcc/beagle/pull/142

**Comment by allanbolipata** [comment]
`10640` imported correctly, but `10720` did not. It may require another fix.

**Comment by allanbolipata** [comment]
Fixed in #143!
